client/js/widgets/widget.js

```diff
@@ -1496,14 +1496,21 @@ export class Widget extends StateManaged {
           const turn = indexList[mod(nextTurnIndex, indexList.length)];
 
           collections[a.collection] = [];
-          //saves turn into all seats and creates output collection with turn seats
-          for(const w of c) {
-            await w.set('turn', w.get('index') == turn);
-            if(w.get('turn') && w.get('player'))
-              collections[a.collection].push(w);
+
+          if(!a.readOnly) {
+            //saves turn into all seats and creates output collection with turn seats
+            for(const w of c) {
+              await w.set('turn', w.get('index') == turn);
+              if(w.get('turn') && w.get('player'))
+                collections[a.collection].push(w);
+            }
+            jeLoggingRoutineOperationSummary(`changed turn of seats from ${previousTurn} to ${turn} - active seats: ${JSON.stringify(indexList)}`);
+          } else {
+            collections[a.collection] = c.filter(w=>w.get('index')==turn && w.get('player'));
```

Comment:
Should the `if` statement in the other branch be changed to this? They appear to do the same thing.

Reply:
done